Core/CoreFactory.swift

```diff
@@ -42,19 +42,17 @@ public enum CoreFactory {
     ///   - tokenHelper: APITokenManagable generated by makeApiHelper method
     ///   - deviceId: Current device UDID set up by UIDeviceInterface protocol
     ///   - reachabilityHelper: ReachabilityChecking generated by makeReachablitiyChecker
-    /// - Returns: GraphQLDispatching protocol
-    public static func makeGraphQLDispatcher(
+    /// - Returns: GraphQLManageable protocol
+    public static func makeGraphQLManager(
         configuration: CoreConfiguration,
         tokenHelper: APITokenManagable,
         deviceId: String,
         reachabilityHelper: ReachabilityChecking
```

Comment:
I notice this reachability helper isn't used any more, and never actually was. Please remove the argument here. 

- [ ] Remove reachabilityHelper